### ISSUE TITLE
[bitnami/golang] 1.19.11-debian-11-r1 misses `go/doc` package.

### ISSUE THREAD

**Issue by kyontan** [bitnami/containers#41060]
### Name and Version

bitnami/golang:1.19.11-debian-11-r1

### What architecture are you using?

amd64

### What steps will reproduce the bug?

check the go's library directory.

```console
$ docker run bitnami/golang:1.19.11-debian-11-r0 ls /opt/bitnami/go/src/go | grep doc
doc
$ docker run bitnami/golang:1.19.11-debian-11-r1 ls /opt/bitnami/go/src/go | grep doc
```

### What is the expected behavior?

The build of the application using `go/doc` succeeds.

### What do you see instead?

The build failures.

```console
$ go build (some program depends on go/doc)
...
/opt/bitnami/go/src/go/build/build.go:13:2: package go/doc is not in GOROOT (/opt/bitnami/go/src/go/doc)
```

### Additional information

`go/doc` module is contained in Go 1.19.11, and should be included under `$GOROOT`.

https://pkg.go.dev/go/doc@go1.19.11

```console
$ docker run golang:1.19.11 ls /usr/local/go/src/go | grep doc
doc
```

**Comment by gongomgra** [comment]
Hi @kyontan,

Thanks for using Bitnami and for reporting this issue! We have found a fix for it and will release a new image including the fix. We will keep you posted.

**Comment by gongomgra** [comment]
Hi @kyontan,

We released a new revision of Golang 1.19 image that should fix the issue: `bitnami/golang:1.19.11-debian-11-r2`.

```
$ docker run --rm -it --entrypoint=/bin/bash bitnami/golang:1.19.11-debian-11-r2
root@9dc708820240:/go# ls /opt/bitnami/go/src/go/doc/
Makefile  comment.go       doc.go       example.go                example_test.go  filter.go    reader.go    synopsis_test.go
comment   comment_test.go  doc_test.go  example_internal_test.go  exports.go       headscan.go  synopsis.go  testdata
```

We are still in the process of releasing an update Golang 1.20 image. Hope the updated 1.19 image works for you, and sorry for the inconveniences!

**Comment by kyontan** [comment]
Hi @gongomgra

Our team have confirmed that the new image solved the issue both in Go 1.19 and 1.20.

Thanks for your quick help and resolution!
